ci/requirements-py35.yml

```diff
@@ -11,6 +11,7 @@ dependencies:
   - pandas
   - seaborn
   - scipy
+  - rasterio
```

Comment:
Unless rasterio only supports Python 3, should add it to other test suites, too (especially Python 2.7).